plutus-ledger/src/Ledger/Index.hs

```diff
@@ -378,24 +372,26 @@ mkTxInfo tx = do
     slotCfg <- pSlotConfig . vctxParams <$> ask
     txins <- traverse mkIn $ Set.toList $ view inputs tx
     let ptx = TxInfo
-            { txInfoInputs = txins
-            , txInfoOutputs = txOutputs tx
-            , txInfoMint = txMint tx
-            , txInfoFee = txFee tx
-            , txInfoDCert = [] -- DCerts not supported in emulator
-            , txInfoWdrl = [] -- Withdrawals not supported in emulator
-            , txInfoValidRange = TimeSlot.slotRangeToPOSIXTimeRange slotCfg $ txValidRange tx
-            , txInfoSignatories = fmap pubKeyHash $ Map.keys (tx ^. signatures)
-            , txInfoData = Map.toList (tx ^. datumWitnesses)
-            , txInfoId = txId tx
+            { txInfoInputs          = txins
+            , txInfoReferenceInputs = mempty      -- FIXME
```

Comment:
`txInfoReferenceInputs` and `txInfoRedeemers` are new fields. I haven't thought yet what goes in here.